apps/studio/components/ui/Forms/FormHeader.tsx

```diff
@@ -21,12 +18,8 @@ const FormHeader = ({
   return (
     <div className={cn(`mb-6 flex items-center justify-between gap-x-4 ${className}`)}>
       <div className="space-y-1">
-        <h3 className="text-foreground text-xl">
-          <ReactMarkdown unwrapDisallowed disallowedElements={['p']}>
-            {title}
-          </ReactMarkdown>
-        </h3>
-        {description && <Markdown content={description} className="max-w-full" />}
+        <h3 className="text-foreground text-xl prose">{title}</h3>
+        {description && <div className="prose text-sm max-w-full">{description}</div>}
```

Comment:
For this one have you checked that we're not passing markdown to this component?

Reply:
Yes - just plain text for both title and description